index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+// TASK 3
+//  1) style.
+
+require_once 'paint-collector-app.php';
+$db = returnDb();
+
+$query = $db->prepare ('SELECT `brand`, `name`, `color`, `base`, `quantity_left`, `purchase_date` FROM `paints`');
+$result = $query->execute();
+$table = $query->fetchAll();
```

Comment:
Would be a good idea to put this in its own function - things could get messy as you add more database queries.

Reply:
Completed. 😸